nifi-nar-bundles/nifi-marklogic-bundle/NOTICE.txt

```diff
@@ -0,0 +1,2106 @@
+MarkLogic NiFi Processors
+
+
+Copyright � 2018 MarkLogic Corporation.
+
+This project is licensed under the Apache License, Version 2.0 (the "License"); you may not use this project except in compliance with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+
+Please direct questions, comments and requests to fossreview@marklogic.com. 

```

Comment:
This is not appropriate for a NOTICE within an Apache project.  Same applies to the above statement.  If these are important to Marklogic it is important to consider that you don't have to contribute your code/Nar to the apache project.  You could simply cancel the PR and JIRA and have this code available on your own company website/etc..

Finally, consider that all contributions to Apache projects are as individuals.  This is not a contribution of Marklogic the corporation or else we cannot accept it.